<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ECVT</class>
 <widget class="QWidget" name="ECVT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="ECVT_StackWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="SMEE::OutputWidget" name="ECVT_Output"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SMEE::OutputWidget</class>
   <extends>QWidget</extends>
   <header>OutputWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
